lua/starfall/libs_sh/darkrp.lua

```diff
@@ -319,18 +369,18 @@ else
 		btnDecline:Dock(LEFT)
 		btnDecline:SetText("Decline")
 		btnDecline:SetAutoSize(false)
-		btnDecline:SetWidth(select(2, self:GetSize())/2)
+		btnDecline:SetWidth(w*0.5)
 		function btnDecline.DoClick()
-			RunConsoleCommand("sf_moneyrequest", 0, "decline", index, expiry)
+			RunConsoleCommand("sf_moneyrequest", 0, "decline", receiver:EntIndex(), expiry)
```

Comment:
Maybe steamid should be used instead

Reply:
Nvm, the request will be deleted if the receive becomes invalid anyway so it'll be fine.